Framework/DataHandling/inc/MantidDataHandling/DefaultEventLoader.h

```diff
@@ -128,7 +128,7 @@ void DefaultEventLoader::makeMapToEventLists(
   } else {
     // To avoid going out of range in the vector, this is the MAX INDEX that can
     // go into it
-    eventid_max = static_cast<int32_t>(pixelID_to_wi_vector.size());
+    eventid_max = static_cast<int32_t>(pixelID_to_wi_vector.size() - 1);
```

Comment:
This change is responsible for the test failure `LoadEventNexusTest::test_load_event_nexus_sans2d_ess`